packages/javascript/bh-shared-ui/src/views/PrivilegeZones/PrivilegeZones.test.tsx

```diff
@@ -32,6 +32,18 @@ vi.mock('react-router-dom', async () => ({
 }));
 
 const server = setupServer(
+    rest.get('/api/v2/features', async (_req, res, ctx) => {
```

Comment:
Thank you for updating these! We can make this more concise and DRY by reusing the zoneHandlers here.


```suggestion
    const server = setupServer(...zoneHandlers);
```

Also for some reason there are two of the following blocks in this test file (not related to your updates):

```
beforeAll(() => server.listen());
afterEach(() => server.resetHandlers());
afterAll(() => server.close());
```

Can we please remove the extra one closer to the top of the file 🙏